images/router/haproxy/conf/haproxy-config.template

```diff
@@ -269,6 +269,12 @@ backend be_edge_http:{{$cfgIdx}}
   balance {{ if gt $cfg.ActiveServiceUnits 1 }}roundrobin{{ else }}leastconn{{ end }}
       {{- end }}
     {{- end }}
+    {{- with $whiteList := index $cfg.Annotations "haproxy.router.openshift.io/whitelist" }}
+      {{- if (matchPattern "([0-9]?[0-9]?[0-9].){3}[0-9]?[0-9]?[0-9](/[0-9]?[0-9]?[0-9])?( ([0-9]?[0-9]?[0-9].){3}[0-9]?[0-9]?[0-9](/[0-9]?[0-9]?[0-9])?)*" $whiteList) }}
```

Comment:
I'd like a tighter regex to match IPs (perhaps load into a variable at the top with the "with" statement of the go template so it is not repeated everywhere)